src/loaders/expressLoader.ts

```diff
@@ -0,0 +1,38 @@
+import express from "express";
+import { Container } from "typedi";
+import { Logger } from "winston";
+
+import config from "../config";
+import apiV1 from "../api";
+import logRequest from "../api/middlewares/logRequest";
+
+
+export const expressLoader = () => {
+    const app = express();
+    const logger: Logger = Container.get("logger");
+
+    app.use(express.json());
+    app.use((req, res, next) => {
+        const timer = logger.startTimer();
+
+        res.on("close", () => {
+            timer.done({ path: req.url });
+        })
```

Comment:
you can use `morgan` library for logging request/response. I'm not sure that the `close ` event covers 100% cases

Reply:
Done.